packages/evolution-backend/src/models/migrations/20240205101500_addLevelToAuditTbl.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import { Knex } from 'knex';
+
+const tableName = 'sv_audits';
+const columnName = 'level';
+
+export async function up(knex: Knex): Promise<unknown> {
+    return knex.schema.alterTable(tableName, (table: Knex.TableBuilder) => {
+        table.enu(columnName, ['error', 'warning', 'info']).defaultTo('error').notNullable().index();
```

Comment:
Before dropping the is_warning column, you should set the level column to 'warning' if is_warning is true

Reply:
Not needed since only one survey uses audits v2 (od_mtl_2023) and audits needs to be reset anyway since they are not yet complete.